enterprise_access/apps/api/v1/views/content_assignments/assignments_admin.py

```diff
@@ -219,24 +221,27 @@ def cancel(self, request, *args, **kwargs):
         tags=[CONTENT_ASSIGNMENT_ADMIN_CRUD_API_TAG],
         summary='Cancel all assignments for the requested assignment configuration.',
         request=None,
+        filters=filters.LearnerContentAssignmentAdminFilter,
```

Comment:
explicitly tell drf-spec what the allowed query params are for this (this view is a POST, so drf-spec wants to look in the request body by default).